### PR TITLE
Dist online fixes

### PR DESCRIPTION
**Pull request by cawallin**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -101,16 +102,20 @@ release: clean
 	python setup.py sdist upload -r pypi_internal
 	python setup.py bdist_wheel upload -r pypi_internal
 
-release-builds: clean-build clean-pyc dist dist-online docs
+release-builds: clean-build clean-pyc dist dist-offline docs
 
 dist: clean-build clean-pyc
```
Comment (ebd2) [inline Makefile]: This should just be:
```
dist: dist-online
```
unless I'm missing something.

Comment (cawallin) [inline Makefile]: yeah, that should work; didn't modify it from before

**Comment by rschlussel** [comment]
looks good.  I assume you tested it.

**Comment by ebd2** [comment]
LGTM

**Pull request merged**